test/tools/router_check/test/route_tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+# router_check_tool binary path
+if [ -z $1 ]
```

Comment:
`-z "$1"` here and elsewhere in the file. See also https://engdoc.corp.google.com/eng/doc/shell.xml?external=y&showone=Quoting&cl=head#Quoting for the rules here.